vault/changelog.release.2021-11-23.md

```diff
@@ -0,0 +1,106 @@
+---
+id: Ku1qjbDYlsxk4rfMoFEza
+title: "0.70"
+desc: ''
+updated: 1637697853678
+created: 1637687416614
+---
+
+Dendron 0.70 has sprouted  🌱
+
+Dendronites can now use `Dendron: Initialize Workspace` to create [[native workspaces|dendron.topic.workspace.native]]. This means that you can now use Dendron inside an existing project by turning one of its folders into your Dendron workspace. For developers, this takes [Docs-as-Code](https://www.writethedocs.org/guide/docs-as-code/) to a whole new level!
+
+This release also includes a variety of fixes and enhancements to publishing, among other improvements.
+
+### Highlights
+- feat(workspace): Initialize Workspace command can create native workspaces ([[docs|dendron.topic.workspace.native]])
+- enhance(publish): Better formatting of tables on published site
+- fix(publish): syntax highlighting for code blocks
+- fix(publish): make mermaid work consistently on published sites
+- fix(publish): enable katex on published site 
+
+### Everything Else
+- enhance(markdown): expose `desc` frontmatter property for variable substitution ([[docs|dendron.topic.markdown#frontmatter-variable-substitution]]) ([diff](https://github.com/dendronhq/dendron-site/pull/284/files))
+- enhance(publish): better position sidebar, main content and footer
+- enhance(commands): copy a tag note link results in hashtags
+- enhance(lookup) Add description field to lookup buttons
+- fix(cli): workspace info prints message to CLI
+- fix(lookup): have schema exact match suggestion in lookup show up at the top of the list
+- fix(lookup): disappearing vaults in vault selection quickpick
+- fix(publish): remove duplicate CSS
+- fix(workspace): remove trailing whitespace in note
+- fix(cli): [ajv](https://github.com/ajv-validator/ajv) (a JSON schema validator) no longer prints warning messages to console
+- fix(pods): invalid configuration error
+
+### Community
+
+As part of the [Dendron Site Reorganization](https://github.com/dendronhq/dendron/discussions/1665), our developer documentation has been migrated over to [docs.dendron.so](https://docs.dendron.so/). With this, Dendronites should notice some slimming of the docs on [wiki.dendron.so](https://wiki.dendron.so/).
+
+#### General Updates
+
+New GitHub discussions to get involved in:
+- [Getting Started Guide: Review, community feedback, and refresh](https://github.com/dendronhq/dendron/discussions/1756)
+- [Cross-linking across workspaces](https://github.com/dendronhq/dendron/discussions/1761)
+
+Blog posts:
+- [Highlights from VSCode 1.62](https://blog.dendron.so/notes/V2Cjla9vzM69Z280j5bXB/)
+
+#### Dendron Reading Series
+
+This week's entry in the [[Dendron Reading Series|community.reading-series]].
+
+![[community.reading-series.journal.2021.11.23]]
+
+#### Office hours
+
+We will be hosting Office Hours every other Wednesday, with these events being created on our [Luma Calendar](https://lu.ma/community/com-lTfMsAZEWSwLJJL/calendar). The next Office Hours will be:
```

Comment:
can you note that this is a change in our office hours? eg. "We are updating our office hour schedules!"